WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html

```diff
@@ -6,32 +6,37 @@
     </head>
 
     <body>
-        <h1>Your Selection</h1>
-        <div class="detail-block-1">
-            <div class="detail-block-1-1">
-                <img src="{% get_static_prefix %}{{ item.image }}" alt="item image">
-                <div class="detail-block-1-1-pane">
-                    <div class="row-1">{{item.name}}</div>
-                    <div class="row-2">
-                        <div id="price-div" data-price="{{ item.price }}">{{item.price}}</div>
-                        <div id="count-square" class="square"><a id="clicks">1</a></div>
-                        <button id="plus-butn" onClick="onClickPlusOne()">+</button>
-                        <button id="minus-butn" onClick="onClickMinusOne()">-</button>
-                    </div>
-                    <div class="row-3">
-                        <p id="before-total-text">TOTAL:&#65129;<span id="total">{{item.price}}</span></p>
-                    </div>
-                    
-                    <div class="row-4">
-                        <a href='' target="_self">PAY NOW</a>
+        <div id="container">
+            <h1>Your Selection</h1>
+            <div class="detail-block-1">
+                <div class="detail-block-1-1">
+                    <img src="{% get_static_prefix %}{{ item.image }}" alt="item image">
+                    <div class="detail-block-1-1-pane">
+                        <div class="row-1">{{item.name}}</div>
+                        <div class="row-2">
+                            <div id="price-div" data-price="{{ item.price }}">{{item.price}}</div>
+                            <div id="count-square" class="square"><a id="clicks">1</a></div>
+                            <button id="plus-butn" onClick="onClickPlusOne()">+</button>
+                            <button id="minus-butn" onClick="onClickMinusOne()">-</button>
+                        </div>
+                        <div class="row-3">
+                            <p id="before-total-text">TOTAL:&#65129;<span id="total">{{item.price}}</span></p>
+                        </div>
+                        
+                        <div class="row-4">
+                            <a href='javascript:onClickPayBtn()' target="_self">PAY NOW</a>
+                        </div>
                     </div>
                 </div>
             </div>
+            <div class="detail-block-2">
+                {{item.description}}
+            </div>
         </div>
-        <div class="detail-block-2">
-            {{item.description}}
+        <div id="overlay">
+            <h2>Please make the payment</h2>
+            <img src="../../static/coffeemachine/images/cashless_payment-transparent.png" alt="alt-image">
```

Comment:
../.. should not be necessary here

Reply:
Changed in commit # [5bbe665](https://github.com/EsamSharaf/WebCoffeeMachine/pull/5/commits/5bbe66577be561bd2edcc364876a1bca47fa1838)